pkg/kubectl/resource/visitor.go

```diff
@@ -201,7 +204,7 @@ func ValidateSchema(data []byte, schema validation.Schema) error {
 		return fmt.Errorf("error converting to YAML: %v", err)
 	}
 	if err := schema.ValidateBytes(data); err != nil {
```

Comment:
It does nothing but return nil. See https://github.com/kubernetes/kubernetes/blob/master/pkg/api/validation/schema.go#L53.